generateapis.sh

```diff
@@ -153,6 +153,8 @@ generate_api Google.Cloud.Spanner.V1 google/spanner/v1 spanner.yaml
 generate_api Google.Cloud.Spanner.Admin.Database.V1 google/spanner/admin/database/v1 spanner_admin_database.yaml
 generate_api Google.Cloud.Spanner.Admin.Instance.V1 google/spanner/admin/instance/v1 spanner_admin_instance.yaml
 
+generate_api Google.Cloud.Bigtable.V2 google/bigtable/v2 bigtable.yaml
```

Comment:
Done. Do you want me to remove the blank above the spanner apis as well while I'm at it?